tests/posix/common/src/clock.c

```diff
@@ -55,15 +55,13 @@ void test_posix_realtime(void)
 	struct timespec rts, mts;
 	struct timeval tv;
 
-	printk("POSIX clock set APIs\n");
-
 	/* Minimal sleep to align us to the next tick interval. This
 	 * helps with a case that 2 consecutive calls to clock_gettime()
 	 * below return different values. Note that it's still a workaround,
 	 * which may break, in which case follow the suggestion in the
 	 * comment above.
 	 */
-	k_usleep(1);
+	k_usleep(101);
```

Comment:
why 101 and not 100?

Reply:
I'll try 100 again to confirm. IIRC I needed to add the delta. And yes, for now it is for ehl_crb, it could happen this way for other fast boards that we add to the project?